ignite/engine/__init__.py

```diff
@@ -3,6 +3,13 @@
 from ignite.engine.engine import Engine, State, Events
 from ignite.utils import convert_tensor
 
+__all__ = [
```

Comment:
Do we need to add it into __init__  too ?

Reply:
We should, since sometimes we also define functions within the `__init__` and import packages, which are actually not part of the API, but required for some checks or something. So this will give us a clean definition, of what to import.